.github/workflows/developer.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: install-dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install libaec-dev libpng-dev zlib1g-dev libjpeg-dev doxygen libopenjp2-7-dev valgrind
+        sudo apt-get install libaec-dev libpng-dev zlib1g-dev libjpeg-dev doxygen libopenjp2-7-dev valgrind libaec-dev
```

Comment:
libaec-dev is listed first and last here.

Reply:
Duh! I will fix.